react/typings/vtex.render-runtime.d.ts

```diff
@@ -1,6 +1,6 @@
 /* Typings for `render-runtime` */
 declare module 'vtex.render-runtime' {
-  import { Component, ComponentType, ReactElement, ReactType } from 'react'
+  import { ComponentType, ReactElement, ElementType } from 'react'
```

Comment:
You can use the render-runtime exported types instead of defining them

Reply:
removed the file, this app doesn't use `render-runtime` and the typings inside this file weren't correct anyway